packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -358,8 +365,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
         l2Sender = _tx.sender;
 
         bool success;
-        (address token,) = gasPayingToken();
-        if (token == Constants.ETHER) {
```

Comment:
While i do like the readability changes here, this adds an extra external call to the system config in the hot code path